src/main/java/core/basesyntax/Figure.java

```
@@ -0,0 +1,14 @@
+package core.basesyntax;
+public class Figure implements Drawable {
+    private Color color;
+    public Figure(Color color) {
+        this.color = color;
+    }
+    public Color getColor() {
+        return color;
+    }
+    public double getArea() {
+        return 0;
+    };
+    public void draw() {};
```

This violates the checklist item: "Don't use abstract classes to set behavior for classes" (the checklist warns you to prefer interfaces and keep behavior separate). `public double getArea() { return 0; }` and `public void draw() {}` are placeholder implementations that can hide errors. Make the behavior explicit by either removing these methods from Figure and using interfaces, or making Figure `abstract` and these methods `abstract` so each concrete shape must implement them.